Neos.Neos/Classes/Controller/Module/Management/WorkspacesController.php

```diff
@@ -122,10 +122,13 @@ protected function initializeAction()
      */
     public function indexAction()
     {
-        $currentAccount = $this->securityContext->getAccount();
-        $userWorkspace = $this->workspaceRepository->findOneByName(UserUtility::getPersonalWorkspaceNameForUsername($currentAccount->getAccountIdentifier()));
-        /** @var Workspace $userWorkspace */
+        $userWorkspace = $this->workspaceRepository->getPersonalWorkspaceByOwnerIdentifier(
+            $this->persistenceManager->getIdentifierByObject(
+                $this->userService->getCurrentUser()
+            )
+        );
 
+        /** @var Workspace $userWorkspace */
```

Comment:
&lt;nitpick&gt;not new with this change but this could be moved to above the line where it's initialized (125)&lt;/nitpick&gt;